network-api/networkapi/wagtailpages/templates/wagtailpages/bannered_campaign_page.html

```diff
@@ -6,3 +6,5 @@
 {% block heroGuts %}
   {% include "./fragments/primary_heroguts.html" with root=root page=page %}
 {% endblock %}
+
+{% block subheader %}{% endblock %}
```

Comment:
this name is a little too generic (some parent of a parent might at some point introduce a `subheader` and then this will cause super fun problems =D), let's rename it to something far more specific, like `campaign_page_header`.

Reply:
✅